Webhooks guide 1: Discussion bot powered by BLOOM (#627)
huggingface/hub-docs: docs/hub/webhooks.md

You will see the HTTP status code and the payload of past events. You can replay those events by clicking on the `replay` button!

When changing the target URL or secret of a webhook, replaying an event will send the payload to the updated URL.

## FAQ

> ##### Can I define webhooks on my organization vs. on my user account?

Reviewer: ```suggestion
##### Can I define webhooks on my organization vs my user account?
```

> ##### Can I define webhooks on my organization vs. on my user account?

Reviewer 2: I couldn't understand the question here, can you specify? e.g. Can I define webhooks on my organization?
(is it like you define webhooks defined from your org, affecting your user repositories?)

Reviewer 2: `Can I define webhooks on my organization's settings like I can with my user's account?` does it sound good 😅

> ##### Can I define webhooks on my organization vs. on my user account?

Reviewer 2: ```suggestion
##### Can I define webhooks on my organization's settings like I can with my user's account?
```

No, this is not currently supported.

> ##### How can I subscribe to events on all repos (or across a whole repo type, like on all models)

Reviewer: ```suggestion
##### How can I subscribe to events on all repos (or across a whole repo type, like on all models)?
```
